scripts/tech-debt-burndown-report.ts

```diff
@@ -1,15 +1,15 @@
 /**
- * TODO(STENCIL-446): Remove this script once `strictNullChecks` is enabled
+ * A script for formatting a Markdown report for CI on some quantities we're tracking as we work
+ * on tech debt on the current branch vs on main. The report includes some info about scritNullChecks errors,
```

Comment:
```suggestion
 * on tech debt on the current branch vs on main. The report includes some info about strictNullChecks errors,
```

Reply:
I should really code with `setlocal spell` on more 🤦‍♀️